package.json

```diff
@@ -64,5 +64,8 @@
     },
     "resolutions": {
         "changelogithub@0.12.7": "patch:changelogithub@npm:0.12.7#.yarn/patches/changelogithub-npm-0.12.7-72f348805d.patch"
+    },
```

Comment:
@saleel We have kept dependencies in specific apps so far, even if they are shared among many projects. Do we want to do the same with siwe?

Reply:
Oh true! Let me fix that.